src/macho.h

```diff
@@ -0,0 +1,42 @@
+/* SPDX-License-Identifier: MIT */
+
+#ifndef MACHO_H
+#define MACHO_H
+
+#include "types.h"
+
+#define MACHO_COMMAND_UNIX_THREAD 0x05
+#define MACHO_COMMAND_SEGMENT_64  0x19
+
+struct macho_header {
+    u32 irrelevant[5];
+    u32 cmdsize;
+    u32 irrelevant2[2];
+};
+
+struct macho_command {
```

Comment:
I'd make these PACKED to be safe in case some compiler decides to introduce additional padding between the members